modules/microshift-exposed-audit-ports-loadbalancer.adoc

```diff
@@ -0,0 +1,49 @@
+// Module included in the following assemblies:
+//
+// * microshift_networking/microshift-networking.adoc
+
+:_mod-docs-content-type: PROCEDURE 
+[id="microshift-exposed-audit-ports-loadbalancer_{context}"]
+
+== NodePort and LoadBalancer service
+
+OVN-Kubernetes opens host ports for `NodePort` and `LoadBalancer` service types. These services add iptables rules that take the ingress traffic from the host port and forwards it to the clusterIP. Logs for the `NodePort` and `LoadBalancer` services are presented in the following examples:
+
+.Procedure 
+
+* You can access the `NodePort` and `LoadBalancer` services logs by running the following command:
++
+[source,terminal]
+----
+$ oc logs -n openshift-ovn-kubernetes ovnkube-master-n2shv ovnkube-master | grep -E "OVN-KUBE-NODEPORT|OVN-KUBE-EXTERNALIP"
```

Comment:
Similar note as above.  The name will be different. To discover the correct name to use 

Maybe a variable could be created using the following command

`oc get pods -n openshift-ovn-kubernetes | awk '/ovnkube-master/{print $1}'`